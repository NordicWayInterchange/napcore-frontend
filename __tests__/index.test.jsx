import { render, screen } from "@testing-library/react";
import Home from "../src/pages/index";
import "@testing-library/jest-dom";

describe("Home", () => {
  it("renders a heading", () => {
    render(<Home />);

    expect(
      screen.getByRole("heading", {
        level: 1,
      })
    ).toHaveTextContent("Hello World");

    // const heading = screen.getByRole("heading", {
    //   name: /welcome to next\.js!/i,
    // });

    // expect(heading).toBeInTheDocument();
  });

  it("renders Home unchanged", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
