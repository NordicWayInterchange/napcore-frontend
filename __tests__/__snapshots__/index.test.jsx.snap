// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders Home unchanged 1`] = `
<div>
  <main>
    <h1
      class="text-3xl text-green-600 p-2"
    >
      Hello World
    </h1>
  </main>
</div>
`;
